red-hat-data-services/odh-manifests
# Add system metrics to model-monitoring

- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-7004

TESTING INSTRUCTIONS:
- Install rhods image on the cluster : `quay.io/vedantm/rhods-operator-liv
e-catalog:1.23.0-7004`
- Go to NS `redhat-ods-monitoring` -> routes -> open route for `rhods-model-monitoring`
- In the prometheus UI, verify if the metrics added in the PR show up as you start typing the metric name. eg: `container_cpu_usage_seconds_total`
- Verify all metrics added in this PR are available in the prometheus UI.
